virtualenvapi/manage.py

```diff
@@ -92,22 +87,28 @@ def _errorfile(self):
         """Absolute path of the log file for recording installation errors."""
         return os.path.join(self.path, 'build.err')
 
+    @property
+    def _virtualenv(self):
+        """The arguments used to call virtualenv."""
+
+        return [sys.executable, '-m', 'virtualenv']
```

Comment:
Because now we can be sure that `virtualenv` is installed into the same environment as the current package, we can call it directly. Otherwise it is possible to receive some strange caused by wrong PATH value or too old virtualenv package.